fast64_internal/f3d/f3d_texture_writer.py

```diff
@@ -517,7 +517,7 @@ def writeAll(
             )
 
         # Write loads
-        loadGfx = fMaterial.material
+        # loadGfx = fMaterial.mat_only_DL # I don't know what the point of this was
```

Comment:
I didn't remove this, but commented it out because I'm not really sure what the point of this was, but it didn't seem like it was doing anything important.

Reply:
Fixed, since loadGfx was derivable from fMaterial I didn't need to be passing it as a parameter to the function. I resolved this the opposite way that you did.